src/components/molecules/LoadingPage/svg/diamond.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="80" height="133" viewBox="0 0 80 133" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All of the svg files should be moved to `assets`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2005/commits/a172c44fbf91deb2b63b290bcdc476c820b80fc1